source/features/mark-unread.tsx

```diff
@@ -158,11 +165,13 @@ function getNotification(notification: Notification): Element {
 		</a>
 	);
 
+	const renderIcon = type === 'issue' ? stateIcons.issue[state as IssueNotificationState] : stateIcons['pull-request'][state];
```

Comment:
This seems clean to me. I went in the spirit if something does not exist, the extension should break. In that regards I omitted doing a switch case with a default that would throw in favor of a ternary of the only known notification types. Does that work for you @bfred-it?